### DISCUSSION TITLE
求大佬帮忙集成Captcha，比如AJ-Captcha。小弟水平差集成一周都没有OK

### DISCUSSION THREAD

**Discussion by lifg1103** [vbenjs/vue-vben-admin#3454] [Q&A]
前端使用Vben-admin，想要参考AJ-Captcha集成到自己项目。水平差没有集成好。
后端已集成OK，就差前端集成组件，并放到登录页面实现**滑块验证或者选文字验证**了。
求**大佬有空的时候怜惜**下，可以直接集成到Vben-admin中，也可以给我提供个直接把文件放在vben-admin后就能直接用的

**Comment by 2919807875** [comment]
look look
